app/src/main/java/com/techcourse/controller/RegisterController.java

```diff
@@ -13,7 +13,7 @@
 public class RegisterController {
 
     @RequestMapping(value = "/register", method = RequestMethod.POST)
-    public ModelAndView save(final HttpServletRequest req, final HttpServletResponse res) throws Exception {
+    public ModelAndView save(HttpServletRequest req, HttpServletResponse res) {
         final var user = new User(2,
```

Comment:
final var를 떼는 것으로 방향을 맞추었다면 일관성 있게 지켜주어도 좋을 것 같습니다.

```suggestion
        User requestUser = new User(2,
```

Reply:
[반영] [6e5d967](https://github.com/woowacourse/java-mvc/pull/835/commits/6e5d96724204c1b13c9928fe50adc9782388ec35)